resources/seccomp/x86_64-unknown-linux-musl.json

```diff
@@ -0,0 +1,1880 @@
+{
+    "vmm": {
+        "default_action": "trap",
+        "filter_action": "allow",
+        "filter": [
+            {
+                "syscall": "epoll_ctl"
+            },
+            {
+                "syscall": "epoll_pwait"
+            },
+            {
+                "syscall": "exit"
+            },
+            {
+                "syscall": "exit_group"
+            },
+            {
+                "syscall": "open"
+            },
+            {
+                "syscall": "read"
+            },
+            {
+                "syscall": "write"
+            },
+            {
+                "syscall": "close"
+            },
+            {
+                "syscall": "brk",
+                "comment": "Called for expanding the heap"
+            },
+            {
+                "syscall": "clock_gettime",
+                "comment": "Used for metrics and logging, via the helpers in utils/src/time.rs. It's not called on some platforms, because of vdso optimisations."
+            },
+            {
+                "syscall": "connect",
+                "comment": "Needed for vsock"
+            },
+            {
+                "syscall": "fstat",
+                "comment": "Used for drive patching & rescanning, for reading the local timezone"
+            },
+            {
+                "syscall": "ftruncate",
+                "comment": "Used for snapshotting"
+            },
+            {
+                "syscall": "lseek",
+                "comment": "Used by the block device"
+            },
+            {
+                "syscall": "mremap",
+                "comment": "Used for re-allocating large memory regions, for example vectors"
+            },
+            {
+                "syscall": "munmap",
+                "comment": "Used for freeing memory"
+            },
+            {
+                "syscall": "recvfrom",
+                "comment": "Used by the API thread and vsock"
+            },
+            {
+                "syscall": "rt_sigreturn",
+                "comment": "rt_sigreturn is needed in case a fault does occur, so that the signal handler can return. Otherwise we get stuck in a fault loop."
+            },
+            {
+                "syscall": "accept4",
+                "comment": "Called by the api thread to receive data on socket",
+                "args": [
+                    {
+                        "arg_index": 3,
+                        "arg_type": "dword",
+                        "op": "eq",
+                        "val": 524288,
```

Comment:
Would it make sense to have these as hex ?

Reply:
unfortunately, JSON does not support hex numbers

Reply:
Hex as string?

Reply:
hmm this would add some extra code complexity (not much), but I don't see a benefit really. WDYT?

Reply:
we synced offline with the team and decided that the benefit of adding hex values is not worth the extra complexity, for now

Reply:
Ack. Makes sense to keep it on radar as an eventual improvement if customers really need it.